manta-accounting/src/wallet/mod.rs

```diff
@@ -72,7 +98,7 @@ where
     checkpoint: S::Checkpoint,
 
     /// Signer Connection
-    pub signer: S,
+    pub signer: S, // This has been made public because of compatibility issues with sdk.
```

Comment:
Make this a function access instead if we need `&mut S` access. And put a safety warning that you can't violate invariants by doing this.